broker-daemon/orderbook/orderbook-index.js

```diff
@@ -64,7 +68,7 @@ class OrderbookIndex {
    * Create a hook for new events added to the store to modify the orderbook
    */
   _addIndexHook () {
-    this.eventStore.pre((dbOperation, add) => {
+    this._removeHook = this.eventStore.pre((dbOperation, add) => {
```

Comment:
naming is confusing. was not expecting `pre` to return this function. Per our talk, would totally appreciate having this code be more clear